recipes-support/provisioning/files/phytec-board-config.sh

```diff
@@ -354,16 +363,12 @@ The next steps are:
  2) Verify your email
  3) Login to your IoT Device Suite account
  4) Check the state of your device in
-    your IoT Device Suite account" $WT_HEIGHT $WT_WIDTH
-		fi
-	else
-		if [ $# -eq 0 ]; then
-			set_onboarded tokendevice
+ your IoT Device Suite account")
```

Comment:
Check the indentation here. It is different from the version before this change. Should have 4 spaces, if I see it correctly.

Reply:
Ignoring for now.